### PR TITLE
DEV-43: changes to deploy api on onrender

### PR DESCRIPTION
**Pull request by mpark63**
## changes 
- in sso.js, callback url after SAML authentication defaults to new onrender.com deployment 
- added new /api/test route that render.com can check to see if deployment is live and healthy 

## todos 
- merge current PR into backend dev and this [PR](https://github.com/uCredit-Dev/ucredit_frontend_typescript/pull/387) into frontend development branch 
- test for a week 
- merge dev into backend main and development into frontend master

ucredit-dev: [https://ucredit-dev.onrender.com](https://ucredit-dev.onrender.com) is a deployment of the [DEV-43](https://github.com/uCredit-Dev/uCredit-API/tree/DEV-43) branch 
ucredit-api: [https://ucredit-apii.onrender.com](https://ucredit-api.onrender.com) is a deployment of the [main](https://github.com/uCredit-Dev/uCredit-API/tree/main) branch

### CODE REVIEW

**Review by mattliu-mygit — APPROVED** [review]
pog awesome!!!!!

File: routes/experiments.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
 const express = require("express");
 const ExperimentDao = require("../data/ExperimentDao");
 const ApiError = require("../model/ApiError");
+const { returnData } = require("./helperMethods.js");
 
 const router = express.Router();
 const experiments = new ExperimentDao();
 
+router.get("/api/test", (req, res) => {
```
Comment (mattliu-mygit) [inline routes/experiments.js:9]: Maybe some documentation on what this is for. Or more stuff added in for testing.

**Review by mattliu-mygit — CHANGES_REQUESTED** [review]
See comment :3

File: routes/experiments.js (hunk 1 above)
Comment (QinyuanWu) [reply routes/experiments.js:12]: if this route is to test whether the backend is online, it should just be the root route "/", and the returned message could be something like "Welcome to uCredit backend" or "uCredit backend alive"

**Review by QinyuanWu — CHANGES_REQUESTED** [review]
requested change to "api/test", see file comments

**Pull request merged**